server/routes/journeys/record-review/check-answers/controller.ts

```diff
@@ -0,0 +1,45 @@
+import { NextFunction, Request, Response } from 'express'
+import { format } from 'date-fns'
+import { BaseJourneyController } from '../../base/controller'
+import { components } from '../../../../@types/csip'
+
+export class ReviewCheckAnswersController extends BaseJourneyController {
+  GET = async (req: Request, res: Response) => {
+    req.journeyData.isCheckAnswers = true
+
+    res.render('record-review/check-answers/view', {
+      review: req.journeyData.review,
+      closingCsip: req.journeyData.review!.outcome === 'CLOSE_CSIP',
+      csipRecordUrl: `/csip-records/${req.journeyData.csipRecord!.recordUuid}`,
+    })
+  }
+
+  checkSubmitToAPI = async (req: Request, res: Response, next: NextFunction) => {
+    const review = req.journeyData.review!
+    try {
+      const action = review.outcome! as components['schemas']['CreateReviewRequest']['actions'][number]
+      await this.csipApiService.createReview(req, {
+        nextReviewDate: review.nextReviewDate!,
+        actions: [action],
+        summary: review.summary!,
+        ...(action === 'CLOSE_CSIP' ? { csipClosedDate: format(new Date(), 'yyyy-MM-dd') } : {}),
+        recordedBy: res.locals.user.username,
+        recordedByDisplayName: res.locals.user.displayName,
+        attendees: (review.attendees || []).map(attendee => ({
+          contribution: attendee.contribution!,
+          isAttended: attendee.isAttended!,
+          name: attendee.name!,
+          role: attendee.role!,
+        })),
```

Comment:
casting it somewhat defeats the purpose of using TS.
here we know exactly what we need to assert for the types to match and can know for sure we didn't mis-cast it.

i'd say we might even assert `review.attendees!` instead of doing `|| []` since we know for sure attendees should be instantiated and populated.